documents/registerNach.md

```diff
@@ -0,0 +1,645 @@
+  "notes": {
+    "notes_key_1": "Tea, Earl Grey, Hot",
+    "notes_key_2": "Tea, Earl Grey… decaf."
+  }
+}
+Razorpay::Customer.create(para_attr)
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| name*          | string      | Name of the customer                        |
+| email        | string      | Email of the customer                       |
+| contact      | string      | Contact number of the customer              |
+| fail_existing | string | If a customer with the same details already exists, the request throws an exception by default. Possible value is `0` or `1`|
+| notes         | object      | A key-value pair                            |
+
+**Response:**
+```json
+{
+  "id": "cust_1Aa00000000003",
+  "entity": "customer",
+  "name": "Gaurav Kumar",
+  "email": "Gaurav.Kumar@example.com",
+  "contact": "9000000000",
+  "gstin": null,
+  "notes": {
+    "notes_key_1": "Tea, Earl Grey, Hot",
+    "notes_key_2": "Tea, Earl Grey… decaf."
+  },
+  "created_at": 1582033731
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Create Order
+
+```rb
+para_attr = {
+  "amount": 100,
+  "currency": "INR",
+  "method": "nach",
+  "receipt": "Receipt No. 5",
+  "notes": {
+    "note_key 1": "Beam me up Scotty",
+    "note_key 2": "Tea. Earl Gray. Hot."
+  },
+  "token": {
+    "first_payment_amount": 10000,
+    "auth_type": "physical",
+    "max_amount": 50000,
+    "expire_at": 1634215992,
+    "notes": {
+      "note_key 1": "Tea, Earl Grey… decaf.",
+      "note_key 2": "Tea. Earl Gray. Hot."
+    },
+    "bank_account": {
+      "beneficiary_name": "Gaurav Kumar",
+      "account_number": 11214311215411,
+      "account_type": "savings",
+      "ifsc_code": "HDFC0001233"
+    },
+    "nach": {
+      "form_reference1": "Recurring Payment for Gaurav Kumar",
+      "form_reference2": "Method Paper NACH",
+      "description": "Paper NACH Gaurav Kumar"
+    }
+  }
+}
+Razorpay.Order.create(para_attr)
+```
+
+**Parameters:**
+
+| Name            | Type    | Description                                                                  |
+|-----------------|---------|------------------------------------------------------------------------------|
+| amount*   | integer      | The amount to be captured (should be equal to the authorized amount, in paise) |
+| currency*   | string  | The currency of the payment (defaults to INR)  |
+| customerId*   | string      | The id of the customer to be fetched |
+| method*      | string  | Payment method used to make the registration transaction. Possible value is `nach`.  |
+| receipt      | string  | Your system order reference id.  |
+| token  | object  | All keys listed [here](https://razorpay.com/docs/api/recurring-payments/paper-nach/auto-debit/#112-create-an-order) are supported |
```

Comment:
auth_type is mandatory .check other [parameters](https://razorpay.com/docs/api/payments/recurring-payments/paper-nach/auto-debit/#111-create-a-customer)

Reply:
added 
https://github.com/razorpay/razorpay-ruby/blob/update_readme/documents/registerNach.md?plain=1#L93